assets/js/utils/rtcUtils.ts

```diff
@@ -184,3 +159,124 @@ export const leaveWebrtc = (webrtc: MembraneWebRTC, clientName: string, webrtcCh
   removeStream(clientName);
   webrtc.leave();
 };
+
+export const shareScreen = async (
+  webrtc: MembraneWebRTC,
+  clientName: string,
+  playerCallback: (SourceType: SourceType) => void
+) => {
+  mergedScreenRef.refreshId && removeMergedStream();
+  const mergedStream = await getMergedTracks(clientName);
+
+  mergedStream.getTracks().forEach((track) => {
+    addOrReplaceTrack(clientName, track, playerCallback);
+  });
+
+  const newTrack = findTrackByType(clientName, "video");
+
+  if (!webrtc || !newTrack) return;
+  if (sourceIds["video"]) webrtc.replaceTrack(sourceIds["video"], newTrack);
+  else sourceIds["video"] = webrtc.addTrack(newTrack, presenterStreams[clientName]);
```

Comment:
Could we simply use here `addOrReplaceTrack`?

Reply:
Those are two different functionalities